ui/partials/perf/graphsctrl.html

```diff
@@ -78,6 +78,7 @@
         </p>
         <p ng-hide="tooltipContent.revision">Loading revision...</p>
         <p id="tt-t" class="small" ng-bind="tooltipContent.date"></p>
+        <p id="tt-v" ng-show="tooltipContent.retriggered > 0">Retriggered: {{tooltipContent.retriggered}}</p>
```

Comment:
Similarly here, wondering about `Retriggers:` and `.retriggers` for these, at least as far as how they read in the tooltip (present tense-number vs. past tense). I defer to @wlach though.

Nice job @MikeLing !